ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessesResourcesSpec.scala

```diff
@@ -787,12 +787,9 @@ class ProcessesResourcesSpec extends FunSuite with ScalatestRouteTest with Match
 
   private def fetchSampleProcess(): Future[CanonicalProcess] = {
     fetchingProcessRepository
-      .fetchLatestProcessVersion[DisplayableProcess](getProcessId(processName))
+      .fetchLatestProcessDetailsForProcessId[CanonicalProcess](getProcessId(processName))
```

Comment:
the same as above. do we need more expensive operation here?

Reply:
It's only tests? And doesn't matter for performance?

Reply:
LOL, right